keys.json

```diff
@@ -0,0 +1,802 @@
+    "890df660ce0ada0388765f9bd4640ed4fca2d601a659b33cf756bfee9f07e8a9606b9c41f0cc3ee6c006b12cc3ee270da9ce081ed3af1ffe9819eec510d013f65c84d990e21ba4466b480b122fd3817269ee0169b9815f5fd1b3d2460559ac8e",
+    "6e1532398394d719e2e7e843ef00b1db8b1180ac529f40b847501128ca3195a66b583b7bea45bde40611409108b8ea058b15bebcba79d31c32a5f2dfd056c683f12e56bf5c25345ba1b8bc04bdc62bed1b7c31e72c78023aff044c765113e287",
+    "218fd8897e2efbf3da6807ff7646885e7338caf248502aa27076a47e55767583327b8d6ebca6e09ebfd8a3ede9b1d90e618c75484f716b4f66958be5a29be31f7ddffabae4fd4e28a65d3954cf2deb20141dd2bfec710ab484d349c3440f6898",
+    "afec6593b12e89a54ce65389dbb19646ccf22e64fb99a6e5c1c617d5e7103547a5326a48895e541969603cef6214191825584d38c55de8a264d0ee696b81edd2da7c11c12eef23fb4707c47aeca7b9d57318d4438cc2f48927c916e07a946111",
+    "dfa170596f4f1d8b9cc299140737db78b679837f5959809d4fbd5ee01eb5ba0f01ff60960a419e9e5fbd3757229e4a172c0de4ba29e033b523cfb55229f852a5378347149883271c880a808572f02eeb628050b3070491617364675092985d94",
+    "b3fb92abebf40780e23e10689b2c66e9e30927d577c28c2d6bc7c4b24d347ee2699c846a23e3e5f826fc93e4d3f63d09f3d90e16f38e1d5c7e30d6a5e3e7efeab0d0ef96f0ec635b78ee3f9caf162bca0b3b4e8ad755642aac17cff020ef5499",
+    "702684b08a88fbd14d79ae6cb63d2bf05459db76feb4b85740c42a167dfc4f5669cc3eb7e3eca91ae3e1baef4a2be00682af8c0fdcc9e33f15871e0fdbbec9fb12fda76e6a4a8b22e6c8e975260db46fb610b33520a3d893ece46d1c3b27dd15",
+    "079344b0c4a635537e394d3c82ac90b3da47dfcae6e7b406dde3b9bec558174dba6ce79ecfd53a467a5dc5843e010f16e767f12d07aed91a066f1694e40024721433c098ea34a1d6abc50078e4ecc2bb5ca98a307e3b03145b3a804f1df77b94",
+    "f720da9751d647c3dbfd6b04ca1f0ed08d8edbc3f558b4be0ad6e259f9f00b9518d86809a4c903b7a2efd81dbed74906f55a2df1a066a1f384e5a76dbd0bddda6840439d4f06c1dda4d9c47f67369f2742399b91dc903d56d3cb7b675633dc04",
+    "afac9551938b9e5aed05c43ed0596cd38b6b0b70d868142117654c1e2426c2e6565a0d201ffdaa4abbd06a8e55001304c757a0f0c46b2bad346c65cc2bdcf11405628e2493625127416dc574946ed93e2cc07a315397f3881229980feea18016",
+    "378353ae6f071d9ff58ca6a1e882c56542618feca610cc77e6fdacf9e1cefcb50dd53d77653d45b96cb57d93200c6201b9cf289f2b85b153d4324dfb725b63774f225d2ecb06720c06851e989b61a05f3e3a160b9ca262880a707718b4456902",
+    "da720180d721eb6e1504844515aa5c9c7cfbc44484d69a9d9bd51702f790eb650efc00df2b3469364cbb2756dedaa111a061c67d8f59765ad1990a5645fc8ed5e0f026dfc948549f1c4aee295338c661d5a3318ce1790e93b57798213cdeed0f",
+    "995a225b38f6011ec3b76caf072bb3f5bf70b567f43ef9bb8507d47ec35269dc1679a5c3c6d4b86038ff48f1c7e9c20ecb90244b0cd9100083969e31442b0928a4fbfb4ded92d4cf08d56f5f20a34d55a36b1def52ac43b54f2853c8fae38d8c",
+    "d499c0cf2eb195acd1561c7b75040d417274bf3c2404784d1ece3ba25ddc85b1955c544bc0b190941a035c9557e68a075c05123a1ac7e5b37e7dffa0df6fe6a5ba47bc865e75487338d58bc57ef45df8115883bd741416ced4f0ca39629e6c0e"
+]
```

Comment:
empty line + contract address: `erd1qqqqqqqqqqqqqpgqxwakt2g7u9atsnr03gqcgmhcv38pt7mkd94q6shuwt` as the last row

Reply:
Fixed